### ISSUE TITLE
Stats are obviously wrong (C++)

### ISSUE THREAD

**Issue by lemire** [vdaubry/github-awards#190]
I have many more than 1531 stars in my C++ projects...
http://git-awards.com/users/lemire

E.g., a single one of my projects exceeds 6000 starts...
https://github.com/lemire/simdjson

**Comment by estiens** [comment]
Yeah, I think it's been years since anything worked on this site...doesn't seem to be updating anymore

**Comment by lemire** [comment]
I think it should be shut down if it is unmaintained?
